pep-0480.txt

```diff
@@ -236,56 +258,63 @@ End-to-End Signing
 ==================
 
 End-to-end signing allows both PyPI and developers to sign for the metadata
-downloaded by clients.  PyPI is trusted to make uploaded projects available to
-clients (PyPI signs the metadata for this part of the process), and developers
-sign the distributions that they upload to PyPI.
-
-In order to delegate trust to a project, developers are required to submit a
-public key to PyPI.  PyPI takes the project's public key and adds it to parent
-metadata that PyPI then signs.  After the initial trust is established,
-developers are required to sign distributions that they upload to PyPI using
-the public key's corresponding private key.  The signed TUF metadata that
-developers upload to PyPI includes information like the distribution's file
+downloaded by clients. PyPI is trusted to make uploaded projects available to
+clients (PyPI signs for the metadata for this part of the process), and
+developers sign for the distributions that they upload to PyPI.
+
+In order to delegate trust to a project, developers are required to submit an
+OpenID Connect identity or a public key to PyPI. PyPI takes the project's OIDC
+identity or public key and adds it to parent metadata, which PyPI then signs.
+After the initial trust is established, developers are required to sign
+distributions that they upload to PyPI using an ephemeral sigstore key, recorded
+in a certificate in sigstore's certificate authority, or should they so desire,
+with the private key corresponding to their public key. The signed TUF metadata
+that developers upload to PyPI includes information like the distribution's file
 size and hash, which package managers use to verify distributions that are
 downloaded.
 
+Two naive approaches regarding end-to-end signing exist: require signing of any
```

Comment:
Understand this concern, but would remove it from this PR, as it adds an unnecessary prescription. This PR should be solely about the optional use of Fulcio for managing developer keys.